src/smt/set_defaults.cpp

```diff
@@ -254,15 +254,15 @@ void SetDefaults::setDefaultsPre(Options& opts)
       SET_AND_NOTIFY_IF_NOT_USER(
           smt, proofMode, options::ProofMode::FULL, "enabling proofs");
     }
-    // Default granularity is theory rewrite if we are intentionally using
+    // Default granularity is DSL rewrite if we are intentionally using
     // proofs, otherwise it is MACRO (e.g. if produce unsat cores is true)
     if (!opts.proof.proofGranularityModeWasSetByUser
         && opts.proof.proofGranularityMode
-               < options::ProofGranularityMode::THEORY_REWRITE)
+               < options::ProofGranularityMode::DSL_REWRITE)
     {
       SET_AND_NOTIFY(proof,
                      proofGranularityMode,
-                     options::ProofGranularityMode::THEORY_REWRITE,
+                     options::ProofGranularityMode::DSL_REWRITE,
```

Comment:
Do I understand correctly that this change will not change the default for Alethe for now?

Reply:
That is correct, I have made Alethe default to THEORY_REWRITE unless granularity is manually set by the user.

Reply:
Cool!